internal/acctest/config/oauth/openidconnect/policy/oauth_open_id_connect_policy_test.go

```diff
@@ -33,6 +35,33 @@ type oauthOpenIdConnectPoliciesResourceModel struct {
 	reissueIdTokenInHybridFlow  *bool
 }
 
+// This is due to a bug in PingFederate that doesn't allow the OAuth client to set "None" as the OIDC Policy
```

Comment:
Could you provide a little more detail in this comment? The actual issue comes up when you try to delete the oidc policy, right?